File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] ret = new int[52];
        for(char c : my_string.toCharArray()){
            if(Character.isLowerCase(c)) ret[c - 'a' + 26] += 1;
            else ret[c - 'A'] += 1;
        }
        return ret;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
from collections import Counter
def solution(my_string):
    cnt = Counter(my_string)
    alp = [0] * 52
    print(cnt)
    for c in cnt.keys():
        if ord(c) < ord('a'):
            alp[ord(c) - ord('A')] = cnt[c]
        else:
            alp[ord(c) - ord('a') + 26] = cnt[c]
    return alp
    